[Worker]: Add test section in getting started guide (#6501)
cloudflare/cloudflare-docs: content/workers/get-started/guide.md

To experiment with more premade Workers, refer to [Workers Examples](/workers/examples/).

## 6. Write test

We recommend writing tests against your worker, one way to do this is with the [unstable_dev](/workers/wrangler/api/#unstable_dev) API in wrangler which is used for writing unit and end-to-end tests.

> One of the prompts from running the `wrangler init` command is a question asking `will you like to write your first test`, and `which test runner you will like to use?`. If you indicate yes and select either vitest or jest as your test runner,  an `index.test.js` file will be created with the following block of code included in the file: 

Reviewer: ```suggestion
When you run the `wrangler init` command, you will be prompted to answer `will you like to write your first test`, and `which test runner you will like to use?`. If you indicate `yes` and select either `vitest` or `jest` as your test runner,  an `index.test.js` file will be created  and populated with the following code:
```
